src/seedsigner/views/screensaver.py

```diff
@@ -151,8 +149,13 @@ def __init__(self, buttons):
         self.min_coords = (0, 0)
         self.max_coords = (self.logo.size[0], self.logo.size[1])
 
-        self.increment_x = self.rand_increment()
-        self.increment_y = self.rand_increment()
+        # pixels to move per second
+        self.pixels_per_second = 150.0
+        
+        # Direction of movement in x and y
```

Comment:
If I'm reading this right, these vars aren't just the direction but rather a kind of magnitude scalar for each axis.

In which case I don't think "direction" is the right naming convention to use. At the moment "scalar" is the best term I can think of. Or "direction_multiplier". Or "delta_rate_x"? "cur_vector" with x/y component?